Sources/SocksCore/Error.swift

```diff
@@ -70,3 +70,144 @@ public struct Error: ErrorProtocol, CustomStringConvertible {
         return "Socket failed with code \(self.number) [\(self.type)] \"\(getReason())\""
     }
 }
+
+public struct ErrorLookUpTabel{
+    
+    public static func getCorrespondingErrorString(errorCode: Int) -> String {
+        // Dictionary indexed from zero
+        if(errorCode > errorDescriptions.count - 1 || errorCode < 0){
+            return "?"
+        }
+        else{
```

Comment:
nit-pick: please keep the `else` clause on the same line as the closing brace, and add a space before the opening, i.e. `} else {`